imports/plugins/core/ui/client/containers/inventoryBadge.js

```diff
@@ -19,9 +18,8 @@ const composer = (props, onData) => {
 
   // Admins pull variants from the Products collection
   if (Reaction.hasPermission(["createProduct"], Reaction.getShopId())) {
-    const inventoryQuantity = ReactionProduct.getVariantQuantity(variant);
     // Product collection variant
-    if (inventoryManagement && !inventoryPolicy && inventoryQuantity === 0) {
+    if (inventoryManagement && !inventoryPolicy && variant.inventoryAvailableToSell === 0) {
```

Comment:
The server does `<=` and I think you need to in case it drops below zero due to the `$inc`s